octobot_trading/personal_data/orders/order.py

```diff
@@ -484,12 +484,23 @@ def update_quantity_with_order_fees(self, other_order):
             logger.debug(f"Updating chained order quantity with triggering order fees. {fees_str}")
         return True
 
+    async def update_price_if_outdated(self):
+        """
+        Implement if necessary
+        """
+
+    async def _adapt_chained_order_before_creation(self, chained_order):
+        can_be_created = True
+        if chained_order.update_with_triggering_order_fees:
+            can_be_created = chained_order.update_quantity_with_order_fees(self)
+        # ensure price is not outdated
+        await chained_order.update_price_if_outdated()
+        return can_be_created
```

Comment:
Is there any reason to put this method in order.py ? it seems that it doesn't require self

Reply:
indeed, i'll move it to order_util 👍